Sources/Hummingbird/Router/TrieRouter.swift

```diff
@@ -15,10 +15,12 @@
 import HummingbirdCore
 
 /// URI Path Trie Builder
-@_spi(Internal) public struct RouterPathTrieBuilder<Value: Sendable> {
+@_documentation(visibility: internal)
+public struct RouterPathTrieBuilder<Value: Sendable> {
```

Comment:
This shouldn't need to be public. It is only ever used to build a router. I'm not sure why the TrieRouter serialize functions have been made `@inlineable`. They don't need to be. Building a router doesn't need to be optmized., It is only the resolve functions that should be `@inlineable`. By removing the inalienable on the serialize functions this can be make internal again.